package.js

```diff
@@ -1,6 +1,6 @@
 Package.describe({
   "summary": "Server Side Rendering for Meteor with Blaze",
-  "version": "2.1.1",
+  "version": "2.2.0",
```

Comment:
Why wouldn't we bump the version? 2.1.1 is not compatible with Meteor 1.2, and this one is not compatible with earlier versions of Meteor.